visual-regexp-steroids.el

```diff
@@ -46,7 +46,9 @@
 ;;; variables
 
 (defvar vr--command-python-default
-  (format "python %s" (expand-file-name "regexp.py" (file-name-directory load-file-name))))
+  (format "python %s"
+          (shell-quote-argument (expand-file-name "regexp.py"
```

Comment:
This is not the right place to put `shell-quote-argument`. You should rather call it when the process is called:

https://github.com/benma/visual-regexp-steroids.el/blob/a6420b25ec0fbba43bf57875827092e1196d8a9e/visual-regexp-steroids.el#L296
and
https://github.com/benma/visual-regexp-steroids.el/blob/a6420b25ec0fbba43bf57875827092e1196d8a9e/visual-regexp-steroids.el#L315

(like the other arguments that are quoted).

Tip: you can git amend your commit and force push it to the same branch to update an existing pull request. This way, you don't have to open a new PR for every iteration.

Please also drop "for Aquaemacs" from the commit title, as this fix is not about Aquaemacs. Thanks.

Reply:
No, those are the wrong places. `shell-quote-argument` needs to be done before `vr--get-command` (and `vr/command-python`) is constructed otherwise you get something like:
```
(shell-quote-argument (vr--get-command))
"python\\ /Users/nega/Library/Preferences/Aquamacs\\ Emacs/Packages/elpa/visual-regexp-steroids-20170222.253/regexp.py"
```
which is obviously incorrect.